### PR TITLE
Styled the scrollbar in homepage

### LINKED ISSUE #1257: Update the theme of scrollbar to be consistent on other pages
### Description
Update the theme of scrollbar to be consistent on other pages as the homepage and also make it more visible on light theme.

### Screenshots

meshery1
meshery2

### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Faizan711**
**Description**
Styled the scrollbar in the home page with site's primary color.

This PR fixes #1257

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
@Faizan711 

Let's discuss it on the websites call.
Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima), if you would :)

**Review by HarshDobariya79 — COMMENTED** [review]
Are the changes in the Gemfile necessary @Faizan711? If not can you please revert it?

File: Gemfile.lock [hunk 1]
```diff
@@ -11,7 +11,6 @@ GEM
     eventmachine (1.2.7)
     eventmachine (1.2.7-x64-mingw32)
     ffi (1.15.5)
-    ffi (1.15.5-x64-mingw32)
```
Comment (Chadha93) [inline Gemfile.lock:14]: @Faizan711 Please revert this Gemfile.lock change.

File: _includes/home-page.html [hunk 2]
```diff
@@ -65,7 +65,7 @@
     }
 
     ::-webkit-scrollbar-thumb {
-      background-color: var(--color-primary-dark);
+      background-color: #00B39F;
```
Comment (Chadha93) [inline _includes/home-page.html:68]: @Faizan711, please use theme ref color, I see this color is in our brand guide, if it's not in them you can add it and then use it.

**Review by Chadha93 — CHANGES_REQUESTED** [review]

**Comment by Chadha93** [comment]
Hi @Faizan711, this issue is resolved by @Ghat0tkach. You might want to review his PR. Let's close this issue, feel free to take new assignments. Thanks.

**Pull request closed without merging**